languages/go/reference/implementing-a-concept-exercise.md

```diff
@@ -68,6 +68,7 @@ If you have any questions regarding implementing the exercise, please post them
 [docs-concept-exercises]: ../../../docs/concept-exercises.md
 [docs-rationale-for-v3]: ../../../docs/rationale-for-v3.md
 [docs-features-of-v3]: ../../../docs/features-of-v3.md
+[anatomy-of-a-concept-exercise]: https://www.youtube.com/watch?v=gkbBqd7hPrA
```

Comment:
The link is never used in this document.

Reply:
Good catch. The Go document was missing the introduction.